### PR TITLE
Stand alone templates 

### PR DESCRIPTION
**Pull request by DpavlovPavlovD**
Publishing two stand-alone templates referred to in the AWS blog post which is soon to be published on Amazon Partner Network. Once the blog posts are online, we can give the link and merge this PR.

### CODE REVIEW

File: stand-alone templates/laureate-template.html [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+    border-bottom: 1px solid var(--mp-color-primary-100);
+  }
+  
+  .page-nav-button {
+    width: 265px;
+    padding: 0;
+    font-size: 12px;
+  }
+  
+  .additional-information {
```
Comment (Abnandan) [inline stand-alone templates/laureate-template.html:64]: Please remove this internal CSS as we are not using it anywhere in the template.

File: stand-alone templates/laureate-template.html [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+    display: grid; 
+    padding-top: 8px; 
+    grid-template-columns: repeat(2, 1fr); 
+    gap: 10px;
+    width: 50%;
+  }
+  
+  ul {
```
Comment (Abnandan) [inline stand-alone templates/laureate-template.html:72]: Please remove the internal CSS for `ul` and pseudo element `ul li::before` as we are not using it anywhere in the template.

File: stand-alone templates/laureate-template.html [hunk 3]
```diff
@@ -0,0 +1,554 @@
...
+    list-style: none;
+  }
+  
+  ul li::before {
```
Comment (Abnandan) [inline stand-alone templates/laureate-template.html:76]: Please remove the pseudo element on `ul` as we are not using it anywhere in the template.

File: stand-alone templates/laureate-template.html [hunk 4]
```diff
@@ -0,0 +1,554 @@
...
+            ?subject <http://data.nobelprize.org/terms/laureateAward> ?laureateAward.
+            ?laureateAward <http://data.nobelprize.org/terms/motivation> ?motivation.
+            ?laureateAward ^<http://purl.org/dc/terms/hasPart> ?nobel.
+            FILTER(LANG(?motivation)="en")
+          }' template='{{>tpl }}'>
+          <template id='tpl'>
+            {{#each bindings}}
+            <div style='display: flex; flex-direction: row;' >
+              <div style='padding-right: 8px;'>
+                <img height='35' src='/assets/nobel.svg' alt=''>
```
Comment (Abnandan) [inline stand-alone templates/laureate-template.html:152]: Replace the `` with material symbols - `<span class="material-symbols-outlined" style='font-size: 35px;'>award_star</span>`

File: stand-alone templates/nobel-prize-template.html
Comment (Abnandan) [inline stand-alone templates/nobel-prize-template.html:1]: LGTM

File: stand-alone templates/laureate-template.html (hunk 1 above)
Comment (DpavlovPavlovD) [reply stand-alone templates/laureate-template.html:64]: accepted, done

File: stand-alone templates/laureate-template.html (hunk 2 above)
Comment (DpavlovPavlovD) [reply stand-alone templates/laureate-template.html:72]: accepted, done

**Review by Abnandan — APPROVED** [review]

**Pull request merged**